File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Currencies/currencysymbols.rb
module Asposetasksjava
  module CurrencySymbols
    def initialize()
    	# Getting Currency Symbol
    	get_currency_symbol()

    	# Getting Currency Symbol
    	set_currency_symbol()
    end

    def get_currency_symbol()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        puts "Currency Symbol: " + project.get(Rjb::import('com.aspose.tasks.Prj').CURRENCY_SYMBOL).to_string
    end

    def set_currency_symbol()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        project.set(Rjb::import('com.aspose.tasks.Prj').CURRENCY_SYMBOL, "$$")

        project.save(data_dir + "ProjectCurrSymbols.mpp", Rjb::import('com.aspose.tasks.SaveFileFormat').MPP)

        puts "Set currency symbol."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithCurrencies/CurrencySymbols.php
<?php
namespace Aspose\Tasks\WorkingWithCurrencies;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\Prj as Prj;
use com\aspose\tasks\SaveFileFormat as SaveFileFormat;

class CurrencySymbols{

    public static function run($dataDir=null){

        # Getting Currency Symbol
        CurrencySymbols::get_currency_symbol($dataDir);

    	# Getting Currency Symbol
    	CurrencySymbols::set_currency_symbol($dataDir);
    }

    public static function get_currency_symbol($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj=new Prj();
        print "Currency Symbol: " . (string)$project->get($prj->CURRENCY_SYMBOL).PHP_EOL;//.to_string
    }

    public static function set_currency_symbol($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj=new Prj();
        $project->set($prj->CURRENCY_SYMBOL, "$$");

        $saveFileFormat=new SaveFileFormat();
        $project->save($dataDir . "ProjectCurrSymbols.mpp", $saveFileFormat->MPP);

        exit;
        print "Set currency symbol.".PHP_EOL;
    }

}
?>